.travis.yml

```diff
@@ -22,3 +22,6 @@ before_script:
 
 script:
   - vendor/bin/phing
+  - >
+    wget https://github.com/maglnet/ComposerRequireChecker/releases/download/0.2.1/composer-require-checker.phar
+    && php composer-require-checker.phar check --config-file=$PWD/config.json composer.json
```

Comment:
I'd move config.json to the `build/` directory and renamed it to composer-require-checker.json 😊